src/Sylius/Bundle/ApiBundle/Serializer/OrderItemNormalizer.php

```diff
@@ -52,16 +44,20 @@ public function normalize($object, $format = null, array $context = [])
         $context[self::ALREADY_CALLED] = true;
 
         $data = $this->normalizer->normalize($object, $format, $context);
-        /** @var ChannelInterface $channel */
-        $channel = $this->channelContext->getChannel();
 
-        /** @var ProductVariantInterface $variant */
-        $variant = $object->getVariant();
+        if ($this->isAdminApiSection()) {
+            unset($data['originalPrice']);
 
-        /** @var ChannelPricingInterface $channelPricing */
-        $channelPricing = $variant->getChannelPricingForChannel($channel);
+            return $data;
+        }
 
-        $data['originalPrice'] = $channelPricing->getOriginalPrice();
+        if (
+            !isset($data['originalPrice']) ||
+            $data['originalPrice'] === 0 ||
+            $data['originalPrice'] === $data['unitPrice']
```

Comment:
I would extract this conditions to private method to improve readbility a little bit

Reply:
As we have a field on OrderItem, I guess we don't need to serialize this field dynamically and we can even remove this logic and the entire normalizer, especially that we can adjust the serialization to serialize or not the proper field

Reply:
Normalized has been removed